core/src/main/java/org/elasticsearch/cluster/routing/RoutingChangesObserver.java

```diff
@@ -26,7 +26,7 @@
     /**
      * Called when unassigned shard is initialized. Does not include initializing relocation target shards.
      */
-    void shardInitialized(ShardRouting unassignedShard);
+    void shardInitialized(ShardRouting unassignedShard, ShardRouting initializingShard);
```

Comment:
should this name here (and elsewhere) be `initializedShard` to be consistent with the method name?